### PR TITLE
Correction of the texture mapping for the Mesh node

### PR DESCRIPTION
**Pull request by BenjaminHug8**
**Description**
The texture mapping with `Mesh` node was not working properly. This can be seen when using `.obj` files.

I just added an option to flip the `y` axis for the texture mapping as Wren might not be aligned with the convention of `OpenGl`.

`OpenGl` and `.obj` define the `y` axis up.

**Related Issues**
This pull-request fixes issue #3945

### CODE REVIEW

**Comment by ad-daniel** [comment]
does this bugfix concern only `obj` files? By flipping it every time, don't we end up breaking the other [formats](https://cyberbotics.com/doc/reference/mesh) instead? Just wondering

**Comment by BenjaminHug8** [comment]
> does this bugfix concern only `obj` files? By flipping it every time, don't we end up breaking the other [formats](https://cyberbotics.com/doc/reference/mesh) instead? Just wondering

Good question. As `.obj`, `.stl` and `.dae` are the three main formats used, we decided to only support these.
I will update this in the documentation and the code in another PR.

**Comment by omichel** [comment]
> I will update this in the documentation and the code in another PR.

If related, the documentation should be update in the same PR.

**Comment by BenjaminHug8** [comment]
> If related, the documentation should be update in the same PR.

I will do that in this PR then.

**Comment by omichel** [comment]
The documentation was not updated in this PR. Can you please update it? Also the change log should include a line about this change,

**Comment by BenjaminHug8** [comment]
> The documentation was not updated in this PR. Can you please update it? Also the change log should include a line about this change,

Yes of course!

File: docs/reference/mesh.md [hunk 1]
```diff
@@ -11,14 +11,9 @@ Mesh {
 The [Mesh](#mesh) node represents a 3D shape imported from an external file created by a 3D modeling software.
 The [Mesh](#mesh) node can be used either as a graphical or as a collision detection primitive (in a boundingObject).
 Currently, the following formats are supported:
-  - [3D Studio mesh](https://wiki.fileformat.com/3d/3ds) files (.3ds).
-  - [Blender](https://www.blender.org/) files (.blend).
-  - [Biovision Hierarchy](https://en.wikipedia.org/wiki/Biovision_Hierarchy) files (.bvh).
```
Comment (ad-daniel) [inline docs/reference/mesh.md:16]: I don't think BVH should be dropped

**Comment by BenjaminHug8** [comment]
@omichel may I ask you a review when you have time?

**Review by omichel — APPROVED** [review]
Thank you.

**Pull request merged**